eng/pipelines/coreclr/ci.yml

```diff
@@ -24,9 +24,6 @@ trigger:
     - eng/pipelines/libraries/*
     - eng/pipelines/runtime.yml
 
-
-pr: none
```

Comment:
Do we ever want this pipeline to be triggerable?

Reply:
Or is that the outerloop pipeline?

Reply:
Yes, ci.yml corresponds to the "runtime-coreclr outerloop" pipeline. I think we should rename the YAML script to "outerloop.yml", this is an endless cause of confusion.

Reply:
I proposed to rename it before as well. However, if we rename it now, we will need to port that to release/5.0 and release/5.0-rc2 branch.

Reply:
Maybe it's just busywork. I would say let's leave this for a more suitable time when there are no porting concerns.